hyper/http20/connection.py

```diff
@@ -20,6 +20,7 @@
 from .window import FlowControlManager
 from .exceptions import ConnectionError
 
+import hyper.http20.errors as errors
```

Comment:
Yes, forgot to ask you about that one. How can I do a relative import and still call `errors.get_data()`? Or should I create a class and call it as `Errors.get_data()`?